### PR TITLE
[JBPM-9308] EventEmitter is not working in spring boot

### PR DESCRIPTION
**Pull request by elguardian**
Jira:
https://issues.redhat.com/browse/JBPM-9308

### CODE REVIEW

**Comment by elguardian** [comment]
depends on https://github.com/kiegroup/jbpm/pull/1738

**Review by afalhambra — CHANGES_REQUESTED** [review]
There is file change which is not needed in this PR. 

I would add a simple test for the Emitter here.

Other than that, everything looks fine.

**Comment by elguardian** [comment]
Jenkins retest this please.

File: kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/src/main/java/org/jbpm/springboot/services/SpringRegisterableItemsFactory.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import java.util.List;
 import java.util.Map;
 
+import org.jbpm.persistence.api.integration.EventEmitter;
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/src/main/java/org/jbpm/springboot/services/SpringRegisterableItemsFactory.java]: done

**Review by afalhambra — APPROVED** [review]
Nice work @elguardian, just wait for CI to clean.

**Pull request merged**